docs/scRNAseq_basics/import_exo.md

```diff
@@ -0,0 +1,31 @@
+![](images/yourmove.png){width="500"}
+
+
+Now it's your time to shine ! We are going to put into pratice what we 
+have just seen. By using a more complicated use case, you are going to 
+reproduce the whole scRNAseq analysis with Seurat.  
+
+# Dataset test
+
+The dataset for this analysis will be single cell RNAseq from chick embryos 
+from [de Lima *et al*](https://doi.org/10.1038/s41467-021-24157-x). We are 
+going to analyse only single cell of E10 stage of embryos. You can download 
+the dataset at the GEO accession [GSE166981](https://0-www-ncbi-nlm-nih-gov.brum.beds.ac.uk/geo/query/acc.cgi?acc=GSE166981).
```

Comment:
Stricto sensus, this is landing on a multi-part html page. Are you expecting the download of `GSE166981_RAW.tar` ?

Reply:
It's a possibility or you can click on custom and download only mandatory files

Reply:
and then it download an archive called `GSE166981_RAW.tar` with only the stage needed. I hope I better explain it in my tips now.

Reply:
Non 😄 je te dis pourquoi plus bas